services/deployment-agent/Makefile

```
@@ -77,6 +80,13 @@ down: ## Stops and remove stack from swarm
 leave: ## leaves swarm stopping all stacks, secrets in it
 	-docker swarm leave -f
 
+.PHONY: env-subst
+env-subst: 
```

```suggestion
.env: template.env
```
in GNU Make, this means it will check template.env exists. and also it will create/update the .env file .